app/api/show-api/src/test/java/artist/fixture/dto/ArtistResponseDtoFixture.java

```diff
@@ -36,4 +38,31 @@ public static List<ArtistSimpleDomainResponse> artistSimpleDomainResponses(
                 .build())
             .toList();
     }
+
+    public static ArtistSearchPaginationDomainResponse artistSearchPaginationDomainResponse(
+        int limit,
+        boolean hasNext
+    ) {
+        return ArtistSearchPaginationDomainResponse.builder()
+            .data(artistSearchSimpleDomainResponses(limit))
+            .limit(limit)
+            .offset(0)
+            .hasNext(hasNext)
+            .build();
+
+    }
+
+    public static List<ArtistSearchSimpleDomainResponse> artistSearchSimpleDomainResponses(
+        int size
+    ) {
+        return IntStream.range(0, size)
+            .mapToObj(i -> ArtistSearchSimpleDomainResponse.builder()
+                .id(UUID.randomUUID())
+                .name(i + "name")
+                .image(i + "testImage")
+                .name(i + "name")
+                .build()
+            )
+            .toList();
```

Comment:
리뷰 포인트 1

IntStream으로 생성되는 숫자가 중요하게 쓰이지 않는 경우, 다음과 같은 문법으로 스트림을 고정 횟수 반복하도록 생성할 수 있습니다!

```suggestion
        return Stream.generate(() -> ArtistSearchSimpleDomainResponse.builder()
                .id(UUID.randomUUID())
                .name("name")
                .image("testImage")
                .build())
                .limit(size)
                .toList();
```